src/libsync/owncloudpropagator.cpp

```diff
@@ -1025,14 +1030,17 @@ void CleanupPollsJob::slotPollFinished()
     } else if (job->_item->_status != SyncFileItem::Success) {
         qCWarning(lcCleanupPolls) << "There was an error with file " << job->_item->_file << job->_item->_errorString;
     } else {
-        if (!_journal->setFileRecord(job->_item->toSyncJournalFileRecordWithInode(_localPath + job->_item->_file))) {
+        // want to do Propagator::updateMetadata()
+        QString filesystemPath = _localPath + job->_item->_file;
+        if (!_journal->setFileRecord(job->_item->toSyncJournalFileRecordWithInode(filesystemPath))) {
             qCWarning(lcCleanupPolls) << "database error";
             job->_item->_status = SyncFileItem::FatalError;
             job->_item->_errorString = tr("Error writing metadata to the database");
             emit aborted(job->_item->_errorString);
             deleteLater();
             return;
         }
+        // TODO: Is syncfilestatustracker notified somehow?
```

Comment:
This is old unused code.
It is going to be updated by https://github.com/owncloud/client/pull/6614

But anyway, no need to update the syncfilestatustracker because this is not about files from this sync, this happen before the discovery.

Reply:
I was wondering whether `SyncFileStatusTracker::slotItemCompleted` would ever be emitted for a file that is finished only after polling completes. You're saying that's working fine and the comment can be removed?